mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/InMemoryMessageIdManager.java

```diff
@@ -27,13 +27,16 @@
 
 import org.apache.james.mailbox.MailboxManager;
 import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.MailboxSession.SessionType;
```

Comment:
the commit title is a little too vague. It's not just security, it's more accessing with different users.